includes/wikia/api/DWDimensionApiController.class.php

```diff
@@ -2,36 +2,101 @@
 
 class DWDimensionApiController extends WikiaApiController {
 	const LIMIT = 100;
-	const AFTER_DOMAIN = null;
+	const LIMIT_MAX = 20000;
+
+	const WIKI_DOMAINS_AFTER_DOMAIN = null;
+
+	const WIKIS_AFTER_WIKI_ID = -1;
+
+	const DART_TAG_VARIABLE_ID = 938;
 
 	private function getSharedDbSlave() {
 		global $wgExternalSharedDB;
 		return wfGetDB( DB_SLAVE, array(), $wgExternalSharedDB );
 	}
 
-	public function getWikiDomains() {
+	public function getWikiDartTags() {
 		$db = $this->getSharedDbSlave();
 
-		$limit = $db->strencode( $this->getRequest()->getVal( 'limit', static::LIMIT ) );
-		$afterDomain = $db->strencode( $this->getRequest()->getVal( 'after_domain', static::AFTER_DOMAIN ) );
-
-		$dbResult = $db->select(
-			[ 'city_domains' ],
-			[ 'city_id', 'city_domain' ],
-			isset( $afterDomain ) ? [ 'city_domain > "'.$afterDomain.'"' ] : [ ],
-			__METHOD__,
-			[
-				'ORDER BY' => 'city_domain',
-				'LIMIT' => $limit
-			]
+		$limit = min($db->strencode( $this->getRequest()->getVal( 'wiki_limit', static::LIMIT ) ), static::LIMIT_MAX);
+		$afterWikiId = $db->strencode( $this->getRequest()->getVal( 'after_wiki_id', static::WIKIS_AFTER_WIKI_ID ) );
+
+		$variables = WikiFactory::getVariableForAllWikis( static::DART_TAG_VARIABLE_ID, $limit, $afterWikiId );
```

Comment:
Let's use `WikiFactory::getVarIdByName` instead of hard-coding variable ID.

Reply:
done